lib/web/desktop.go

```diff
@@ -193,6 +193,7 @@ func proxyClient(ctx context.Context, sessCtx *SessionContext, addr string) (*cl
 	if err := cfg.ParseProxyHost(addr); err != nil {
 		return nil, trace.Wrap(err)
 	}
+	cfg.HostLogin = sessCtx.user
```

Comment:
Technically `HostLogin` should be the user on the target host, which we have in the `username` variable we pull from the query string on line 92.

It doesn't _really_ matter, as this is only used when using the `ProxyClient` to connect to an SSH node, but we do have more accurate information so we might as well use it.

I would also add a comment here:
```
// HostLogin must be specified in order to avoid defaulting to 
// the local Unix user, which will fail in some environments.
```

Reply:
Edit: I had time and realized it would be several hours before you're awake, so I went ahead and made the change. Let me know what you think!